packages/rocketchat-setup-wizard/server/lib/serverHasAdminUser.js

```diff
@@ -0,0 +1,5 @@
+Meteor.methods({
+	serverHasAdminUser() {
```

Comment:
Can't you check this on client side?

Reply:
No because I'm looking for any existing admin on the server.